frontend/components/transaction/BidTransactionTable.vue

```diff
@@ -0,0 +1,164 @@
+        >
+            <div :class="{ underline: isActive && minimumDepositDai }">Auction minimum bid</div>
+            <div>
+                <format-currency v-if="minimumDepositDai && isActive" :value="minimumDepositDai" currency="DAI" />
+                <div v-else class="opacity-50">Unknown</div>
+            </div>
+        </div>
+        <div class="flex justify-between items-center">
+            <div>The amount to bid</div>
+            <div class="w-1/2">
+                <bid-input
+                    :amount-to-bid.sync="amountToBid"
+                    :minimum-deposit-dai="minimumDepositDai"
+                    :total-price="auctionTransaction.totalPrice"
+                    :disabled="auctionTransaction.isFinished || !auctionTransaction.isActive"
+                />
+            </div>
+        </div>
+        <div class="flex justify-between font-bold">
+            <div>The amount to receive</div>
+            <div>
+                <format-currency
+                    v-if="amountToReceive && isActive && !isBidAmountNaN"
+                    :value="amountToReceive"
+                    :currency="auctionTransaction.collateralSymbol"
+                />
+                <div v-else>
+                    <span class="opacity-50">UNKNOWN</span>
+                    <span class="font-bold">{{ auctionTransaction.collateralSymbol }}</span>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import BigNumber from 'bignumber.js';
+import BidInput from '../utils/BidInput.vue';
+import TimeTill from '~/components/common/TimeTill.vue';
+import FormatCurrency from '~/components/utils/FormatCurrency.vue';
+import PriceDropAnimation from '~/components/utils/PriceDropAnimation.vue';
+
+export default Vue.extend({
+    components: {
+        TimeTill,
+        FormatCurrency,
+        PriceDropAnimation,
+        BidInput,
+    },
+    props: {
+        auctionTransaction: {
+            type: Object as Vue.PropType<AuctionTransaction>,
+            required: true,
+        },
+        minimumDepositDai: {
+            type: Object as Vue.PropType<BigNumber>,
+            required: true,
+        },
+    },
+    data() {
+        return {
+            amountToBid: undefined as BigNumber | undefined,
+        };
+    },
+    computed: {
+        amountToReceive(): BigNumber | undefined {
+            if (!this.amountToBid || !this.auctionTransaction.approximateUnitPrice) {
+                return this.auctionTransaction.collateralAmount;
+            }
+            return this.amountToBid.dividedBy(this.auctionTransaction.approximateUnitPrice);
+        },
+        isActive(): boolean {
+            return this.auctionTransaction.isActive && !this.auctionTransaction.isFinished;
+        },
+        isBidAmountNaN(): boolean {
+            return this.amountToBid?.isNaN() || this.amountToBid?.isEqualTo(0) || false;
```

Comment:
Then you can remove 0 from here
```suggestion
            return this.amountToBid?.isNaN();
```